pkg/cmd/completion.go

```diff
@@ -73,7 +73,6 @@ var completionInstallCmd = &cobra.Command{
 		var buf bytes.Buffer
 		var path string
 		var name string
-
```

Comment:
Erroneous whitespace change

Reply:
this is the whitespace you asked me to remove up here: https://github.com/DopplerHQ/cli/pull/379#discussion_r1158041154

Reply:
Based on the diff, I wouldn't expect this file to change in this PR. My previous comment was noting that whitespace was added. The whitespace is now removed, but so is the entire newline. Once fixed, I would expect this file to be omitted from the diff.